app/upgrade_handler.go

```diff
@@ -414,3 +417,16 @@ func prefixToShentuAddrs(addrs []string) (newAddrs []string, err error) {
 	}
 	return newAddrs, err
 }
+
+func transDescriptionForBank(ctx sdk.Context, app ShentuApp) (err error) {
+	baseKeeper := app.BankKeeper.BaseKeeper
+	metaData := baseKeeper.GetAllDenomMetaData(ctx)
+
+	for _, meta := range metaData {
+		if meta.Description == "The native staking token of the CertiK Chain." {
```

Comment:
Do we need this condition?

Reply:
we can also remove the if directly and assign the value directly

Reply:
https://github.com/shentufoundation/shentu/pull/770#discussion_r1321189765